R/arrange.R

```diff
@@ -59,8 +59,8 @@
 #'   help page under the `Default locale` section.
 #' @family single table verbs
 #' @examples
-#' # Load the {palmerpenguins} package (R < 4.5.0)
-#' library(palmerpenguins)
+#' # Load the 'penguins' data set from {palmerpenguins} package (R < 4.5.0)
+#' data("penguins")
```

Comment:
```suggestion
#' data("penguins", package = "palmerpenguins")
```

No need for a comment if the is more clear 😄